### PR TITLE
add TFM netcoreapp5.0 to N.B.T.Console and N.P.Core projects

### PR DESCRIPTION
**Pull request by heng-liu**
## Bug

Fixes: https://github.com/NuGet/Home/issues/9605
Regression: No  
* Last working version:   
* How are we preventing it in future:   

## Fix

Details: 
The code map changed a little bit after we adding netstandard2.1(now netcoreapp5.0) to NuGet.Packaging related projects.
So adding netcoreapp5.0 TFM for NuGet.Build.Tasks.Console and NuGet.Packaging.Core projects

## Testing/Validation

Tests Added: No  
Reason for not adding tests:  
Validation:

### CODE REVIEW

File: src/NuGet.Core/NuGet.Build.Tasks.Console/NuGet.Build.Tasks.Console.csproj [hunk 1]
```diff
@@ -1,4 +1,8 @@
 <Project>
+  <PropertyGroup>
+  <RequiresSigningXplatAPIs>true</RequiresSigningXplatAPIs>
```
Comment (nkolev92) [inline src/NuGet.Core/NuGet.Build.Tasks.Console/NuGet.Build.Tasks.Console.csproj]: indent items in the prop group.

File: src/NuGet.Core/NuGet.Packaging.Core/NuGet.Packaging.Core.csproj [hunk 2]
```diff
@@ -1,4 +1,8 @@
 <Project>
+  <PropertyGroup>
+  <RequiresSigningXplatAPIs>true</RequiresSigningXplatAPIs>
```
Comment (nkolev92) [inline src/NuGet.Core/NuGet.Packaging.Core/NuGet.Packaging.Core.csproj]: indent items in the prop group.

**Review by nkolev92 — COMMENTED** [review]
Do we need to bump the version for the respective tests too?

File: src/NuGet.Core/NuGet.Build.Tasks.Console/NuGet.Build.Tasks.Console.csproj (hunk 1 above)
Comment (heng-liu) [reply src/NuGet.Core/NuGet.Build.Tasks.Console/NuGet.Build.Tasks.Console.csproj]: Thanks! Fixed.

File: src/NuGet.Core/NuGet.Packaging.Core/NuGet.Packaging.Core.csproj (hunk 2 above)
Comment (heng-liu) [reply src/NuGet.Core/NuGet.Packaging.Core/NuGet.Packaging.Core.csproj]: Thanks! Fixed.

**Comment by heng-liu** [comment]
> Do we need to bump the version for the respective tests too?

Checked and found the N.B.T.Console.**Test**.csproj has already targeted net5.0
https://github.com/NuGet/NuGet.Client/blob/dev-feature-xplatVerification/test/NuGet.Core.Tests/NuGet.Build.Tasks.Console.Test/NuGet.Build.Tasks.Console.Test.csproj#L2-L4

**Review by zivkan — APPROVED** [review]

**Review by nkolev92 — APPROVED** [review]

**Pull request merged**